packages/renderer/MdToHtml/rules/image.ts

```diff
@@ -29,8 +29,7 @@ function plugin(markdownIt: any, ruleOptions: RuleOptions) {
 
 				js = ` ontouchstart="${touchStart}" ontouchend="${cancel}" ontouchcancel="${cancel}" ontouchmove="${cancel}"`;
 			}
-
-			return `<img data-from-md ${htmlUtils.attributesHtml(Object.assign({}, r, { title: title }))}${js}/>`;
+			return `<img data-from-md ${htmlUtils.attributesHtml(Object.assign({}, r, { title: title, alt: alt }))}${js}/>`;
```

Comment:
What happens when alt is not defined or an empty string?

Reply:
When alt or title are not defined it returns `<img data-from-md="" data-resource-id="some-id" src="path to src file" title alt>`